Shoko.Server/API/Startup.cs

```diff
@@ -91,6 +91,8 @@ public void ConfigureServices(IServiceCollection services)
                         var xml = Path.Combine(Path.GetDirectoryName(location), $"{Path.GetFileNameWithoutExtension(location)}.xml");
                         if (File.Exists(xml)) options.IncludeXmlComments(xml); //Include the XML comments if it exists.
                     }
+                    options.MapType<v3.Models.Shoko.SeriesType>(() => new Schema { Type = "string" });
```

Comment:
https://www.newtonsoft.com/json/help/html/JsonConverterAttributeProperty.htm

This mapping stuff is way unnecessary, unless I'm wrong, but I don't think I am

Reply:
If you have a fix for the types showing up wrong in swagger, then please do tell me how to fix it, since that mapping was their recommended approach (from their project read-me).